app/command_utils.cc.template

```diff
@@ -5,77 +5,46 @@
 
 #include "brave/app/command_utils.h"
 
-#include <iterator>
-#include <map>
 #include <string>
-#include <vector>
 
 #include "base/feature_list.h"
-#include "base/no_destructor.h"
-#include "base/ranges/algorithm.h"
-#include "base/strings/string_split.h"
-#include "base/strings/string_util.h"
 #include "brave/app/brave_command_ids.h"
 #include "brave/components/commands/common/features.h"
 #include "chrome/app/chrome_command_ids.h"
-
-#define ADD_UNTRANSLATED_COMMAND(name) \
-  {                                    \
-    IDC_##name, {                      \
-      GetName(#name)                   \
-    }                                  \
-  }
+#include "base/containers/fixed_flat_map.h"
+#include "base/containers/span.h"
 
 namespace commands {
 namespace {
 
-std::string GetName(const std::string& raw_name) {
-  auto words = base::SplitString(raw_name, "_", base::TRIM_WHITESPACE,
-                                 base::SPLIT_WANT_NONEMPTY);
-  for (auto& word : words) {
-    if (word.size() == 1) {
-      continue;
-    }
-    word = word[0] + base::ToLowerASCII(word.substr(1));
-  }
-  return base::JoinString(words, " ");
-}
+constexpr auto kCommandInfos = base::MakeFixedFlatMap<int, base::StringPiece>({
+COMMAND_NAMES
+});
 
-const base::flat_map<int, std::string>& GetCommandInfo() {
-  static const base::NoDestructor<base::flat_map<int, std::string>>
-      kCommands({
-TEMPLATE_PLACEHOLDER
-      });
-  return *kCommands;
-}
+
+constexpr int kCommandIdsRaw[] = {
+COMMAND_IDS
+};
+
+constexpr base::span<const int> kCommandIds(kCommandIdsRaw);
 
 }  // namespace
 
-const std::vector<int>& GetCommands() {
+const base::span<const int>& GetCommands() {
```

Comment:
please pass/return by value all "view" types (such as `base::span`, `base::StringPiece`). It's more effective to just copy them, because they don't contain any data other than a pointer and a size_t.